src/_guides/libraries/library-tour_migrated.md

```diff
@@ -1278,10 +1275,10 @@ Dart language features, see the
 To get each value as it arrives, either use `await for` or
 subscribe to the stream using the `listen()` method:
 
-<?code-excerpt "misc/lib/library_tour/async/stream_web.dart (listen)" replace="/listen/[!$&!]/g"?>
+<?code-excerpt "../null_safety_examples/misc/lib/library_tour/async/stream_web.dart (listen)" replace="/listen/[!$&!]/g"?>
 {% prettify dart tag=pre+code %}
 // Find a button by ID and add an event handler.
-querySelector('#submitInfo').onClick.[!listen!]((e) {
+querySelector('#submitInfo')!.onClick.[!listen!]((e) {
```

Comment:
I wonder if we should have a note about the `!`, since not all readers might be familiar with that syntax, and not all readers might be using null safety yet.

Reply:
In that case, wouldn't it be better to extract the `querySelector` and have something like:

```suggestion
submitButton.onClick.[!listen!]((e) {
```

I tried to imagine how I'd address the `!`, and it seems weird. I'd spend more time explaining the `!` than the topic at hand.

Reply:
Resolved with latest commit